example/algorithms_evaluation/run.py

```
@@ -0,0 +1,194 @@
+import sys
+import os
+import re
+import time
+from multiprocessing import Process
+from shlex import quote
```

Ruff (0.12.2)

`shlex.quote` imported but unused

Remove unused import: `shlex.quote`

(F401)